src/__swaps__/screens/Swap/components/SwapSheetGestureBlocker.tsx

```diff
@@ -13,28 +13,28 @@ export const SwapSheetGestureBlocker = ({
   children: React.ReactNode;
   preventScrollViewDismissal?: boolean;
 }) => {
-  const { configProgress, inputProgress, outputProgress } = useSwapContext();
-  const [enabled, setEnabled] = useState<boolean>(false);
+  // const { configProgress, inputProgress, outputProgress } = useSwapContext();
```

Comment:
is this stuff meant to remain commented out?

Reply:
yeah disabled until https://linear.app/rainbow/issue/APP-1459/implement-swapsheetgestureblocker is in